frame/pallet-xvm/Cargo.toml

```diff
@@ -0,0 +1,56 @@
+[package]
+name = "pallet-xvm"
```

Comment:
Maybe rename the folder to just `xvm` to stay consisten with the other palletes.

Reply:
This is apparently the directory name used upstream...